package.json

```diff
@@ -1,27 +1,34 @@
 {
-  "name": "LiveBand.js",
+  "name": "remote-performer",
   "version": "0.0.0",
   "description": "Real-time music collaboration for the web",
   "main": "src/index.js",
   "scripts": {
-    "start": "node src/index.js",
-    "postinstall": "bower install",
-    "lint": "./node_modules/.bin/standard"
+    "build": "browserify -t brfs src/public/js/app.js > src/public/js/bundle.js",
+    "start": "npm run build && node src/index.js",
+    "lint": "standard"
```

Comment:
You should keep standard in the dev deps. npm "run script" actually shims ./node_modules/.bin into the path when you run it... so if you were to include standard as a dev dep it would resolve to ./node_modules/.bin/standard